models/src/main/thrift/content/v1.thrift

```diff
@@ -886,7 +886,7 @@ struct Tag {
 }
 
 struct Atoms {
-    1: optional contentatom.Atom quiz
+    1: optional contentatom.Atom atom
```

Comment:
Even if this is work, as it does not match the content api response, I think it make thing a bit more complicated:
```
 val quiz = atoms.get.quiz.get
```
will not work if you have both a viewpoint and an quizz.